src/app/public/modules/date-pipe/date.pipe.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  DatePipe
+} from '@angular/common';
+
+import {
+  Pipe,
+  PipeTransform
+} from '@angular/core';
+
+import {
+  SkyAppLocaleProvider
+} from '@skyux/i18n';
+
+import {
+  BehaviorSubject
+} from 'rxjs/BehaviorSubject';
+
+@Pipe({
+  name: 'skyDate',
+  pure: false
+})
+export class SkyDatePipe implements PipeTransform {
+  private get format(): string {
```

Comment:
Sorry, I'm a bit new to pipes. Are these getters and setters supposed to all be private?

Reply:
In this case, yes. A pipe is very similar to an Angular service, and they can be injected into class constructors. For this initial release, however, the values are internal only. We can always make them public in the future, if an issue is filed for a feature request.

Reply:
sounds good. thanks for the explanation.